template/codeland.rb

```diff
@@ -183,6 +188,7 @@ def user_not_authorized
 gem 'autoprefixer-rails', '~> 5.2.1'
 gem 'bootstrap-sass', '~> 3.3.5'
 gem 'font-awesome-rails', '~> 4.4.0'
+gem 'draper', '~> 1.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.